docs/Intro.mdx

```diff
@@ -0,0 +1,43 @@
+![Welcome to the Discord Developer Platform](docs-header.svg)
+
+# Welcome to the Discord Developer Platform
+
+The Discord developer platform lets you develop apps to customize and extend Discord for millions of users. A toolkit of [APIs](#DOCS_REFERENCE) and [resources](#DOCS_TOPICS_COMMUNITY_RESOURCES) pair splendidly with a supportive community to help you build bots, integrate 3rd party services, and add Discord support to your game.
+
+<Button href="https://discord.com/developers/docs/getting-started" color="brand">Get Started</Button>
```

Comment:
Now that I notice it, seems strange to have an absolute url here.

Not sure that it matters too much since none of the links on the site seem to prefetch, it just seems unnecessarily verbose (and a big departure from the established link syntax)

Reply:
Kinda what I thought, thanks for clarifying